_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+id: Custom Resource Definition
+name: Custom Resource Definition
+tags:
+- fundamental
+- operation
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
+short-description: >
+  A custom resource definition is the representation of a customization of a particular Kubernetes installation.
```

Comment:
Something like: `An API object that provides a way of extending the Kubernetes API by defining custom API objects.`?

 Would prefer to use the word "API object" to stay consistent with `Deployment`, `Service`, etc. Also, because it is an API object. :)

Reply:
The trouble with "API object" in this context is that an object is created from a call to a resource. Which, in this case, is a _resource_ created by the CRD. (The core docs page on CRDs is confusing here, and makes a muddle out of "endpoint", "resource", "object".) There's no easy solution here, I'm afraid. Also, short description still needs to be made a fragment.